streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -392,8 +392,12 @@ private[ui] class StreamingPage(parent: StreamingTab)
       maxX: Long,
       minY: Double,
       maxY: Double): Seq[Node] = {
+    val maxYCalculated = listener.receivedEventRateWithBatchTime.map {
+      case (streamId, eventRates) => new EventRateUIData(eventRates).max.getOrElse(0.0)
```

Comment:
This seems good @proflin. You'll need to rebase anyway, so I have one tiny suggestion: you can omit `streamId` here and write `_`. I suppose you can `flatMap` and move the `getOrElse` outside the map, but I don't think this matters.